src/model_io/codecs/include/iDynTree/ModelIO/ModelExporter.h

```diff
@@ -26,9 +26,8 @@ namespace iDynTree
  *
  * Options for the iDynTree exporter.
  */
-class ModelExporterOptions
+struct ModelExporterOptions
 {
-public:
```

Comment:
What is the rationale of this change? On some platforms this could break code that forward declare this class, but if there is a good reason we can change it.

Reply:
Since it was a class w/ only public members I thought a struct is more suitable, but I can undo it if it breaks user code!

Reply:
With a first glance nobody seems to forward declare this at least in public repos, so probaly we can change this without problem.